pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -579,7 +579,9 @@ func warnIfCertificatesExpired(config *igntypes.Config) {
 			data := decoded.Data
 			for {
 				block, rest := pem.Decode(data)
-				if block != nil {
+				if block == nil {
```

Comment:
This section has become somewhat convoluted with all of the breaks. Can we simplify the `for` loop?
```	
for len(data) > 0 {
	block, rest := pem.Decode(data)
	if block == nil {
		break
	}
	data = rest
	// check cert
}
```

Reply:
We actually don't even need the check for the length of the data either, since an empty data will return a nil block from `pem.Decode`.

Reply:
I've simplified this.